ABOUT_THIS_SERVICE.md

```diff
@@ -0,0 +1,20 @@
+# About this service
+
+## Mission Statement
+
+OPG has many 'things' to measure to aid in decision making, but these are not visible to all or accessible in one place, making it difficult to see patterns and connections, or high level problems across products.
+
+## Problems this service aims to solve
+
+* A common language that can be used across teams and OPG.
+* Encourage and Gamify the culture of celebrating maintenance and managing legacy
+* Understanding our users over time
+* Hold ourselves accountable throughout the entire service life-cycle
+* Think about performance and monitoring at the start and what done/good looks like
+* Reduce barriers to entry for getting data
+
+## How should you use this service
+
+This service is not intended to be a place where you debug issues or store logs that you can diagnose problems. It is to enable you to pass key bits of information across multiple services so you can overlay data points, find patterns and maintain visibility of your service.
```

Comment:
Feel like it needs something about how it should be comprehensible to non-technical staff to make decisions or kickstart investigations..